kaizen/cli/commands/utils/confirmation_manager.py

```
@@ -0,0 +1,296 @@
+"""User confirmation management utilities.
+
+This module provides utilities for handling user confirmations during test execution,
+including auto-fix and pull request creation confirmations.
+"""
+
+import os
+from typing import Any, Dict, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)